modules/edu.gemini.seqexec.server/build.sbt

```diff
@@ -0,0 +1,20 @@
+import Settings._
+import Settings.Libraries._
+import Settings.LibrariesJVM._
+
+name := "edu.gemini.seqexec.server"
+
+libraryDependencies ++= Seq(
+  ScalaZCore,
+  ScalaZConcurrent,
+  "io.argonaut"        %% "argonaut"                  % "6.1",
+  "commons-httpclient" % "commons-httpclient"         % "2.0",
+  // OCS bundles
+  SpModelCore,
+  POT,
+  EpicsACM
+)
```

Comment:
I kind of like the simplicity of just writing them all out, except when there are a bunch of modules with the same version in which case just that segment can be factored out. I don't feel strongly about it though.